maxim/dataset/dataset.py

```
@@ -109,3 +116,171 @@ def validate_data_structure(
+            # Make the PATCH call with all updates
+            try:
+                res = self.api._MaximAPI__make_network_call(
+                    method="PATCH",
+                    endpoint="/api/sdk/v4/datasets/entries",
+                    body=json.dumps({
+                        "datasetId": dataset_id,
+                        "updates": updates
+                    }),
+                )
+            except Exception as e:
+                raise Exception(f"Failed to update dataset entries with attachments: {str(e)}") from e
```

Ruff (0.12.2)

Create your own exception

(TRY002)